include/test.h

```diff
@@ -0,0 +1,13 @@
+/*
+ *  * Copyright © 2017-2018 The Crust Firmware Authors.
```

Comment:
Please fix the indentation and duplicated asterisks (hint: `set paste` in `vim` before pasting, then `set nopaste` afterward).

Reply:
done